code/datums/crew_credits/crew_credits.dm

```diff
@@ -4,6 +4,9 @@
 	var/list/crew_tab_data
 	var/list/antagonist_tab_data
 	var/list/score_tab_data
+	var/list/citation_tab_data
+
+	var/has_citation_data = FALSE
```

Comment:
why not just `if length citation tab data [ tickets] || length citation tab data [ fines]`

Reply:
d2fc640